msamp/deepspeed/runtime/config.py

```diff
@@ -26,6 +28,7 @@ def _initialize_params(self, param_dict):
         super()._initialize_params(param_dict)
         self.msamp_enabled = get_msamp_enabled(param_dict)
         self.msamp_optlevel = get_msamp_optlevel(param_dict)
+        self.msamp_usete = get_msamp_usete(param_dict)
```

Comment:
`self.msamp_use_te` or `self.msamp_te_enabled`

Reply:
Changed from `self.msamp_usete` to `self.msamp_use_te`